transforms/universal/tokenization/ray/Dockerfile

```diff
@@ -13,11 +13,9 @@ COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
 RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
 COPY --chown=ray:users data-processing-lib-ray/ data-processing-lib-ray/ 
 RUN cd data-processing-lib-ray    && pip install --no-cache-dir -e .
-COPY --chown=ray:users python-transform/ python-transform 
-RUN cd python-transform && pip install --no-cache-dir -e .
+COPY --chown=ray:users python-transform/ python-transform
+RUN cd python-transform && pip install --no-cache-dir -r requirements.txt && pip install --no-cache-dir -e .
```

Comment:
with recent fixes to .make.defaults, I don't believe `-r requirements.tx` is required anymore.  Others don't seem to need this.

Reply:
I'd like to gradually make it as a practice. When we start supporting Poetry, it will be needed.